scripts/upgrade.sh.in

```diff
@@ -6,17 +6,17 @@ set -o pipefail
 VERBOSE=${VERBOSE:-0}
 LOGFILE=/var/log/metalk8s/upgrade.log
 DRY_RUN=0
-SALTENV=${SALTENV:-}
-DESTINATION_VERSION=""
+DESTINATION_VERSION=${DESTINATION_VERSION:-@@VERSION}
+# SALTENV must be equal to script version and DESTINATION_VERSION
+# (checked by the precheck orchestrate)
+SALTENV="metalk8s-$DESTINATION_VERSION"
 SALT_CALL=${SALT_CALL:-salt-call}
 BASE_DIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
 
 
 _usage() {
     echo "upgrade.sh [options]"
     echo "Options:"
-    echo "--destination-version "
-    echo "   <destination-version>:        Destination version to upgrade to"
```

Comment:
Not really related to this PR, is it? Also, would need to update documentation accordingly.

Reply:
already done, not pushed yet.